execution/executor-test-helpers/src/integration_test_impl.rs

```diff
@@ -183,8 +192,8 @@ pub fn test_execution_with_storage_impl() -> Arc<AptosDB> {
     let txn5 = get_test_signed_transaction(
         account2,
         /* sequence_number = */ 0,
-        privkey2,
-        pubkey2,
+        privkey2.clone(),
```

Comment:
what we'd like is to replace the get_test_signed_transaction with local_account.sign_with_transaction_builder